Tests/airport.spec.js

```diff
@@ -7,16 +7,25 @@ let plane;
 let airport;
 
 //1st Test
-console.log(`--------------------`)
-console.log(`1. addtailNumber adds Plane to the Airport`)
+console.log('--------------------');
+console.log('1. addtailNumber adds Plane to the Airport');
 
 //Arrange
-
+airport = new Airport();
+plane = new Plane();
```

Comment:
Shouldn't have needed to declare Plane class as it should be called by the Airport.